src/python/WMCore/WMSpec/StdSpecs/TaskChain.py

```diff
@@ -400,9 +399,7 @@ def setupTask(self, task, taskConf):
                                               splitArgs = taskConf["SplittingArguments"],
                                               stepType = cmsswStepType,
                                               forceUnmerged = forceUnmerged,
-                                              timePerEvent = self.timePerEvent,
                                               memoryReq = taskConf.get("Memory", None),
-                                              sizePerEvent = self.sizePerEvent, 
```

Comment:
I have no idea. If the exist, then yes, it would make sense to pass those. Should I change the pull request accordingly ?